README.md

```diff
@@ -221,7 +221,7 @@ The both defaults badge (noob) already was added to all users that we already ha
 Inside your application if you want to give 100 points to some user, inside your function you have to use the following method:
 
 ```ruby
-kind =  Kind.where(:name => "teacher")
```

Comment:
@joaomdmoura - It doesn't - that's why it's been removed and replaced with the line below using a `find_by` (the red `-` indicates it's a removal). I've tested this and updated to use the code below to get it to work. Both of the following work (tested):

    kind = Kind.find_by(:name => "teacher")
    kind = Kind.where(:name=>"teacher").first